compiler/passes/ResolveScope.cpp

```diff
@@ -639,8 +640,8 @@ Symbol* ResolveScope::lookupForImport(Expr* expr, bool isUse) const {
           retval = sym;
           break;
         }
-        USR_FATAL(expr, "import must name a module ('%s' not a module)",
-                  sym->name);
+        USR_FATAL(expr, "%s must name a module ('%s' not a module)",
+                  type, sym->name);
```

Comment:
It can be an enum for use statements, but this error message doesn't indicate that